# An array of target poses for multiple legs
LegTarget[] targets
